src/utils/BrambleExtensionLoader.js

```diff
@@ -60,7 +61,8 @@ define(function (require, exports, module) {
     var extraExtensions = [
         "brackets-cdn-suggestions",    // https://github.com/szdc/brackets-cdn-suggestions
         "ImageUrlCodeHints",
-        "HTMLHinter"
+        "HTMLHinter",
+        "Autosave"
```

Comment:
Actually, just tested it. It doesn't need to be in the extraExtensions list to allow disabling.